pandas/core/indexes/multi.py

```diff
@@ -2702,7 +2702,10 @@ def _get_loc_single_level_index(self, level_index: Index, key: Hashable) -> int:
         if is_scalar(key) and isna(key):
             return -1
         else:
-            return level_index.get_loc(key)
+            loc = level_index.get_loc(key)
+            # if isinstance(loc, slice):
```

Comment:
this is commented?

Reply:
Shit, no I tried something out. This has to get back in. Will add it back momentarily

Reply:
Commented it back in. At least my test failed previously :)